src/index.ts

```diff
@@ -24,20 +25,26 @@
 }
 
 async function getPRDetails(): Promise<PRDetails> {
-  const { repository, number } = JSON.parse(
-    readFileSync(process.env.GITHUB_EVENT_PATH || "", "utf8")
-  );
-  const prResponse = await octokit.pulls.get({
-    owner: repository.owner.login,
-    repo: repository.name,
-    pull_number: number,
+  const context = github.context;
+  if (!context.payload.pull_request) {
+    throw new Error("This action can only be run on pull requests");
```

Comment:
The check for `context.payload.pull_request` is a good addition to ensure that the action is only run on pull requests. This prevents potential runtime errors when the action is triggered by other events.